Environment/AirQualityObserved/example.json

```diff
@@ -29,5 +29,5 @@
   "NO2": 69,
   "NOx": 139,
   "SO2": 11,
-  "refPointOfInterest": "28079004-Pza. de España"
+  "refPointOfInterest": "28079004-Pza.deEspanna"
```

Comment:
Although not that important, I have read somethings "España" as "Espanya" (kind of the Catalan way :p) more than "Espanna"... I guess it's because of how it sounds... :)